src/main/webapp/user_script.js

```diff
@@ -1,9 +1,12 @@
-
 function loadProfile() {
-  fetch('/profile-data')
-    .then((response) => response.json())
-    .then((profile) => {
-      var profileContainer = document.getElementById('profile-container');
-      commentElement.innerText = profile;
-    });
+  fetch('/profile-data').then((response) => response.json()).then((profile) => {
+    var nameSection = document.getElementById('name');
```

Comment:
Replace the use of ```var nameSection``` with ```const nameSection```. Make sure to check this section of the style guide so you apply it to the other variables as well: https://google.github.io/styleguide/jsguide.html#features-use-const-and-let